src/web/app/pages-session/session-submission-page/session-submission-page.component.ts

```diff
@@ -90,6 +90,7 @@ export class SessionSubmissionPageComponent implements OnInit, AfterViewInit {
   formattedSessionClosingTime: string = '';
```

Comment:
You do not need to change this file at all. The number of questions can be obtained from the length of `questions: QuestionSubmissionFormModel[]`